### ISSUE TITLE
Xcode 10, Apple Watch (arm64_32) minimum deployment

### ISSUE THREAD

**Issue by jordimele** [CocoaLumberjack/CocoaLumberjack#947]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

`This bundle is invalid. Applications built for more than one architecture require an iOS Deployment Target of 3.0 or later.`

You have to set the deployment target for all watchOS targets (although it says iOS) to 3.0 or higher..

By upload to App Store Connect we have one error to minimumOS deployment.

**Comment by KeimyPlaza** [comment]
Maybe this error is only for Xcode10 but this library must be to supported Xcode 9.X

You can change the podfile:

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['WATCHOS_DEPLOYMENT_TARGET'] = '3.0'
    end
  end
end
```
Reference: https://forums.developer.apple.com/thread/108408

**Comment by sushichop** [comment]
@jordimele 
Thanks for reporting. 
I've re-added the issue template info to the description, as this information is vital to us.

If use CocoaPods for integration, you can set minimum OS versions(iOS 10.0 and watchOS 3.0).
At least I was able to build my app with Xcode 10.0. Podfile is like this.

```ruby
def shared_pods
  pod 'CocoaLumberjack/Swift'
  pod 'CocoaLumberjack/Extensions'
end

target 'AppleWatchSample' do
  platform :ios, '10.0'
  use_frameworks!
  shared_pods
end

target 'AppleWatchSample WatchKit App' do
  platform :watchos, '3.0'
  use_frameworks!
  shared_pods
end

target 'AppleWatchSample WatchKit Extension' do
  platform :watchos, '3.0'
  use_frameworks!
  shared_pods
end
```

**Comment by sushichop** [comment]
@KeimyPlaza 

> Reference: https://forums.developer.apple.com/thread/108408

This info is helpful. Thanks.

**Comment by bpoplauschi** [comment]
I see the problem has been properly answered here. Closing now. Feel free to reopen/continue the conversation if there are remaining issues.
